.gitignore

```diff
@@ -35,3 +36,6 @@ bench/workflows/*
 
 # Annoying Mac file
 *.DS_Store
+
+# Arsen's local server configuration
+conf/arsen*
```

Comment:
where else? I just put this in the gitignore to have a clean working copy when changing my configuration